src/music21/stream.ts

```diff
@@ -1945,13 +1945,73 @@ export class Stream extends base.Music21Object {
         }
     }
 
+    /**
+     * Returns either (1) a Stream containing Elements
+     * (that wrap the None object) whose offsets and durations
+     * are the length of gaps in the Stream
+     * or (2) None if there are no gaps.
+     * @returns {boolean}
+     */ 
+    findGaps(self) {
+        if (!self.isSorted && self.autoSort) {
+            self.sort();
+        }  
+        const sortedElements = self.elements;
+        let eDur = 0;      
+        const gapStream = new Score(); // cloneEmpty does not work, created new obj       
```

Comment:
What happened when you tried to use `cloneEmpty`? Looks like we don't have any tests for this yet.